rectanglesurrounding/input.txt

```diff
@@ -0,0 +1,8 @@
+1
+1 1 2 2
+1
+10 10 19 19
+2
+10 10 19 19
+15 15 24 24
+0
```

Comment:
Let's update the input such that the first line starts with an integer `K` indicate the number of test cases. E.g.: 

```suggestion
3
1
1 1 2 2
1
10 10 19 19
2
10 10 19 19
15 15 24 24
```

So your stub should do a for-loop `K` times for each test cases.